# Cost Benefit Analysis

We did a cost benefit analysis on the assumption of the time period of 2013 where we had sprayed the entire Chicago city (costs) with pesticides in order to prevent any economic productivity loss and hospitalization costs (benefit).

Bassed on our research:

  - The number of [human WNV cases](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7241786/) in Chicago region (refer to Table 2 in the link) <b>in 2013 was 66 cases</b>
  
  
  - The [CDC site](https://www.cdc.gov/westnile/index.html) shares that while there are no vaccines to prevent or medications to treat WNV in people, about <b>1 in 5 people</b> who are infected develop a fever and other symptoms and about <b>1 out of 150</b> infected people develop a serious, sometimes fatal, illness. We will subsequently use this data in our Benefits section to estimate the productivity loss and healthcare costs.

## Costs

Spraying Costs


- [Time of spray](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_communities/news/2020/august/city-to-spray-insecticide-thursday-to-kill-mosquitoes.html): Weather permitting, the spraying will begin at dusk on August 13th and continue through the night until approximately 1:00 am, with licensed mosquito abatement technicians in trucks dispensing an ultra-low-volume spray. The material being used to control the adult mosquitoes, Zenivex.


- [Spray rate of Zerivex](https://chicago.cbslocal.com/2017/08/30/spray-mosquitoes-far-south-side-west-nile-prevention/): The chemical used is Zenivex, applied at a rate of 1.5 fluid ounces per acre. That measure is approved by the U.S. EPA to control mosquitoes in outdoor residential and recreational areas.


- [Zenivex cost per acre](https://www.centralmosquitocontrol.com/-/media/files/centralmosquitocontrol-na/us/resources-lit%20files/zenivex%20cost%20comparison%20fact%20sheet.pdf): USD0.67 per acre



- Area of Chicago City: estimated to be 145,300 acres based on Google Search




Based on the above, we assume the cost of spray to be USD 0.67 per acre multiplied by the area of Chicago city. This accounts for 1 spraying exercise.

In [62]:
# cost of 1 spraying exercise
spray_cost = 145300*0.67
spray_cost

97351.0

The cost of 1 spraying exercise is about USD 97,351.

We assume we want to be aggresive and spray every week during the summer months of July, August and September based on our spray effectiveness analysis in the previous notebook.

In [63]:
# total cost of spraying annually
# assuming each spray * 4 weeks * 3 months
spray_cost * 4 * 3

1168212.0

For this scenario, we would need to set aside a budget of about USD 1.17 million to conduct the spraying exercises on an annual basis.

## Benefits

On the assumption of our aggressive spraying policy, no one contracted the virus. As such, we would have prevented the 66 human WNV cases in Chicago for 2013. 

Had we not sprayed so aggresively, these 66 cases would have resulted in:

- 1 person being hospitalized (due to developing a serious illness)
- The other 65 people infected with mild symptoms

Based on CDC's estimates (1 in 150 people require hospitalization), since [older people are more likely](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4941816/) to contract a more serious illness than younger people, we made the assumption that the 1 person being hospitalized is 60 years old or above.

Medical Health Costs

Patients who were hospitalized with acute flaccid paralysis, a partial- to whole-body paralysis caused by WNV infection, had the largest initial and long-term medical costs ([median USD 25,000 and USD 22,000 respectively](https://www.sciencedaily.com/releases/2014/02/140210184713.htm)).

In [23]:
# estimated initial hospitalization cost and long term medical costs for the 1 person

healthcare_cost = 25000+22000
healthcare_cost

47000

The approximate [economic productivity loss](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322011/) (referenced from Table 3 and Table 4 in the link) is calculated to be:


In [24]:
# Estimated economic costs of WNV cases due to productivity loss 
# for the hospitalized patient and 65 mild symptomatic patients
# proxy from a California study

# 125 * 60 --> productivity loss for the person being hospitalized (aged 60 years old or above)
#              value of each day * number of days

# (955*65)  --> productivity loss for the 65 other people developing mild symptoms
#                           

productivity_loss = (125*60) + (955*65)
productivity_loss

69575

[Inflated adjusted productivity loss](https://www.minneapolisfed.org/about-us/monetary-policy/inflation-calculator/consumer-price-index-1913-) inflated to 2013 value since the California study was based in 2005.

In [25]:
productivity_loss_adjusted = productivity_loss * (1+0.032) * (1+0.029) * (1+0.038) * (1-0.004) * (1+0.016) * (1+0.032) * (1+0.021) * (1+0.015) 
productivity_loss_adjusted

82998.48565928808

In [26]:
total_benefit = healthcare_cost + productivity_loss_adjusted
total_benefit

129998.48565928808

The economic productivity loss is estimated USD 83,000 (inflated adjusted to 2013). Adding onto the healthcare cost, the total monetary benefit would have been about USD 130,000.

We conclude that the prevention of economic productivity loss and healthcare cost of USD 130,000 does not sufficiently justify the spending of USD 1.17 million to spray pesticides.

Based on the analysis above, <b> the costs outweighed the benefits</b> of the spraying in 2013 due to the pathological nature of the WNV which we have studied so far. Furthermore, referencing back to our previous notebook on the spray data effectiveness analysis, the spray effectiveness is very sensitive to weather patterns.